app/src/main/java/com/infomaniak/swisstransfer/ui/components/SwissTransferAlertDialog.kt

```diff
@@ -31,15 +31,36 @@ import com.infomaniak.swisstransfer.ui.theme.Margin
 import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 import com.infomaniak.core2.R as RCore2
 
+object SwissTransferAlertDialogDefaults {
+    @Composable
+    fun confirmButton(isEnabled: () -> Boolean = { true }, onClick: () -> Unit) {
+        SmallButton(
+            style = ButtonType.TERTIARY,
+            title = stringResource(R.string.buttonConfirm),
+            enabled = isEnabled,
+            onClick = onClick,
+        )
+    }
+
+    @Composable
+    fun cancelButton(onClick: () -> Unit) {
+        SmallButton(
+            style = ButtonType.TERTIARY,
+            title = stringResource(RCore2.string.buttonCancel),
+            onClick = onClick,
+        )
+    }
+}
+
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun SwissTransferAlertDialog(
     modifier: Modifier = Modifier,
     @StringRes titleRes: Int,
     @StringRes descriptionRes: Int,
+    positiveButton: @Composable () -> Unit,
+    negativeButton: @Composable () -> Unit,
```

Comment:
This should already use by default your newly created default buttons
I'd suggest to make this nullable all the way to ActionButton, and then manage it there

Reply:
I would've loved to as well, but we need to define the on click callbacks outside of it, so I can't set them as default values. The only option would be to provide two signatures for the `SwissTransferAlertDialog`; one that takes composables and one that takes only onClick callbacks. But this feels a bit complicated :/